doc/user/content/releases/v0.97.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Is this file supposed to be here? Related to `version-added`?

Reply:
Yep, exactly!